external/unity/version.txt

```diff
@@ -0,0 +1 @@
+https://github.com/ThrowTheSwitch/Unity/commit/b4aca70fd9e0ddf0afbdafb1b826f5edcfc1049b
```

Comment:
wouldn't it be better to sync from a release tag?

Reply:
Generally yes, I vaguely remember there was some reason not to use the latest release, but I will check again.